src/annotator/components/test/ContentInfoBanner-test.js

```diff
@@ -19,26 +20,76 @@ const contentInfo = {
   },
 };
 
+const createComponent = props =>
+  mount(<ContentInfoBanner info={contentInfo} {...props} />);
+
 describe('ContentInfoBanner', () => {
-  it('renders banner', () => {
-    const wrapper = mount(<ContentInfoBanner info={contentInfo} />);
-    assert.include(wrapper.text(), contentInfo.item.title);
+  it('shows linked partner logo', () => {
+    const wrapper = createComponent();
+
+    const logoLink = wrapper.find('Link[data-testid="logo-link"]');
+    const logoImg = wrapper.find('img[data-testid="logo-image"]');
+
+    assert.equal(logoLink.prop('href'), contentInfo.logo.link);
+    assert.equal(logoImg.prop('src'), 'https://www.jstorg.org/logo.svg');
```

Comment:
```suggestion
    assert.equal(logoImg.prop('src'), 'https://www.jstor.org/logo.svg');
```

Not that it matters, but I assume this was a typo.